package.json

```diff
@@ -30,63 +31,41 @@
     "babel-core": "^6.26.3",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.2.2",
-    "electron": "^12.0.1",
     "esbuild": "^0.14.27",
     "jest": "^26.6.3",
     "jest-dynalite": "^3.3.1",
     "limiter": "2.1.0",
-    "mysql": "^2.18.1",
-    "starknet": "^4.22.0",
     "nodemon": "^2.0.7",
     "prettier": "^2.2.1",
     "serverless": "^2.31.0",
     "serverless-esbuild": "^1.26.1",
     "serverless-offline": "^7.0.0",
     "serverless-prune-plugin": "^1.4.4",
     "serverless-webpack-fixed": "^5.3.3",
+    "starknet": "^4.22.0",
     "ts-jest": "^26.5.4",
     "ts-loader": "^8.0.18",
     "ts-node": "^9.1.1",
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@acala-network/api": "^3.0.2",
-    "@connext/nxtp-contracts": "0.0.94",
-    "@connext/nxtp-utils": "0.0.94",
     "@defillama/sdk": "^5.0.44",
-    "@makerdao/multicall": "^0.12.0",
-    "@polkadot/api": "^6.7.1",
-    "@port.finance/port-sdk": "^0.1.42",
     "@project-serum/anchor": "^0.18.2",
     "@project-serum/serum": "^0.13.33",
-    "@pythnetwork/client": "^2.6.1",
     "@sentry/serverless": "^6.2.3",
     "@sentry/tracing": "^6.2.3",
     "@solana/web3.js": "^1.30.2",
     "@supercharge/promise-pool": "^2.4.0",
-    "@terra-money/terra.js": "^3.0.7",
-    "@vechain/connex-driver": "^2.0.7",
-    "@vechain/connex-framework": "^2.0.7",
-    "algosdk": "^1.12.0",
     "async-retry": "^1.3.1",
-    "atob": "^2.1.2",
     "axios": "^0.21.0",
-    "axios-rate-limit": "^1.3.0",
     "bignumber.js": "^9.0.1",
-    "borsh": "^0.6.0",
-    "caver-js": "^1.6.5",
     "dotenv": "^8.2.0",
     "ethers": "^5.4.6",
-    "graph-results-pager": "^1.0.3",
     "graphql": "^15.3.0",
     "graphql-request": "^3.2.0",
-    "mysql2": "^2.3.3",
+    "hi-base32": "^0.5.1",
```

Comment:
why add these?

Reply:
forgot for which chain, but  maybe tron helper needs this